api/forgot.go

```diff
@@ -49,10 +51,16 @@ func (a *Api) passwordReset(res http.ResponseWriter, req *http.Request, vars map
 	var resetCnf *models.Confirmation
 	var reseterLanguage string
 
-	// By default, the reseter language will be his browser's or "en" for Englih
+	// on the "forgot password" page in Blip, the preferred language is now selected by listbox
+	// even if not selected there is one by default so we should normally always end up with
+	// a value in GetUserChosenLanguage()
+	// however, just to play it safe, we can continue taking the browser preferred language
+	// or ENglish as default values
```

Comment:
```suggestion
	// or English as default values
```
?

Reply:
this one is on purpose as we actually return "EN" for "English"